compiler/ir/backend.js/src/org/jetbrains/kotlin/ir/backend/js/transformers/irToJs/JsIntrinsicTransformers.kt

```diff
@@ -167,17 +170,17 @@ class JsIntrinsicTransformers(backendContext: JsIrBackendContext) {
             }
 
             for ((type, prefix) in intrinsics.primitiveToTypedArrayMap) {
-                add(intrinsics.primitiveToSizeConstructor[type]!!) { call, context ->
+                add(intrinsics.primitiveToSizeConstructor[type] ?: doError(intrinsics.primitiveToSizeConstructor)) { call, context ->
                     JsNew(JsNameRef("${prefix}Array"), translateCallArguments(call, context))
                 }
-                add(intrinsics.primitiveToLiteralConstructor[type]!!) { call, context ->
+                add(intrinsics.primitiveToLiteralConstructor[type] ?: doError(intrinsics.primitiveToSizeConstructor)) { call, context ->
```

Comment:
I don't think these will ever fail

Reply:
I think it is related to a lot of lines in this file, but I already faced some issues in compiler without adequate message or environment info

Reply:
I'm not convinced. I don't see a reason to add effectively dead code.

Reply:
So, may you guarantee somehow that this code will not be used? :) As I have said, in the whole file were a lot of `!!` which means `it is potential thrown error without any useful info`. I just have added some explanation to these things. In case you have some suggestion better than use `!!` or `?: doError`, I would be glad to see it and even realize, I think